ui-admin/src/study/participants/participantList/ParticipantList.test.tsx

```diff
@@ -29,6 +31,38 @@ test('renders a participant with link', async () => {
   expect(participantLink).toHaveAttribute('href', `/${studyEnvContext.currentEnvPath}/participants/JOSALK`)
 })
 
+test('renders filters for participant columns', async () => {
+  const studyEnvContext = mockStudyEnvContext()
+  const { RoutedComponent } = setupRouterTest(<ParticipantList studyEnvContext={studyEnvContext}/>)
+  render(RoutedComponent)
+
+  //Assert that all 3 default columns have filter inputs
+  await waitFor(() => {
+    expect(screen.getAllByPlaceholderText('Search...')).toHaveLength(3)
+  })
+})
+
+test('filters participants based on shortcode', async () => {
+  const studyEnvContext = mockStudyEnvContext()
+  const { RoutedComponent } = setupRouterTest(<ParticipantList studyEnvContext={studyEnvContext}/>)
+  render(RoutedComponent)
+
+  //Assert that JOSALK is visible in the table
+  await waitFor(() => {
+    expect(screen.getByText('JOSALK')).toBeInTheDocument()
+  })
```

Comment:
`getBy...` and `findBy...` throw if no matching element is found, so this could be simplified to:

```suggestion
  await screen.findByText('JOSALK')
```

Reply:
Done in a281500b